src/components/SearchBar.jsx

```diff
@@ -1 +1,65 @@
-// implement SearchBar component here
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class SearchBar extends React.Component {
+  render() {
+    const { searchText, onSearchTextChange, bookmarkedOnly,
+      onBookmarkedChange, selectedGenre,
+      onSelectedGenreChange } = this.props;
```

Comment:
Tenho que arruma o meu também, mais acho que assim fica bem melhor de ler.
```suggestion
    const {
    searchText,
    onSearchTextChange,
    bookmarkedOnly,
    onBookmarkedChange,
    selectedGenre,
    onSelectedGenreChange,
    } = this.props;
```